vm/ubpf_jit_arm64.c

```diff
@@ -966,14 +973,20 @@ translate(struct ubpf_vm* vm, struct jit_state* state, char** errmsg)
         // All checks for errors during the encoding of _this_ instruction
         // occur at the end of the loop.
         struct ebpf_inst inst = ubpf_fetch_instruction(vm, i);
-        state->pc_locs[i] = state->offset;
 
         if (i == 0 || vm->int_funcs[i]) {
+            size_t prolog_start = state->offset;
             emit_movewide_immediate(state, true, temp_register, ubpf_stack_usage_for_local_func(vm, i));
             emit_addsub_immediate(state, true, AS_SUB, SP, SP, 16);
             emit_loadstorepair_immediate(state, LSP_STPX, temp_register, temp_register, SP, 0);
+            // Record the size of the prolog so that we can calculate offset when doing a local call.
+            if (state->bpf_function_prolog_size == 0) {
+                state->bpf_function_prolog_size = state->offset - prolog_start;
+            }
```

Comment:
I know that this sounds like paranoia, but is there a way that we could verify that the prologue size is always the same (after it is calculated the first time)?